src/processor/AliasProcessor.ts

```diff
@@ -0,0 +1,42 @@
+import fs from 'fs-extra';
+import { ExportableAlias } from '../exportable';
+import { logger } from '../utils/GoFSHLogger';
+
+export class AliasProcessor {
+  static process(aliasFile: string): ExportableAlias[] {
+    const aliases: ExportableAlias[] = [];
+
+    // Load aliases from alias-file option.
+    if (this.isProcessableAliasFile(aliasFile)) {
+      let aliasFileContent;
+      try {
+        aliasFileContent = fs.readFileSync(aliasFile, 'utf8');
+      } catch (e) {
+        logger.warn(`Alias file read failed with error: ${e.message}`);
+      }
+      if (aliasFileContent != null) {
+        const aliasFileContentLines = aliasFileContent.split('\n');
+        for (let i = 0; i < aliasFileContentLines.length; i++) {
+          const aliasFileContentLine = aliasFileContentLines[i];
```

Comment:
```suggestion
          const aliasFileContentLine = aliasFileContentLines[I].trim();
```
Calling `trim()` on each line as we pull them out will allow for Aliases to still be detected if a user includes a leading space.